src/components/Layout/Layout Grid/LayoutGrid.vue

```diff
@@ -0,0 +1,124 @@
+<template>
+  <section class="layout-grid-container pb-row">
+    <div class="pb-row pb-col-12 pb-col-md-3">
+      <div class="sidebar pb-col-12">
+        <div class="pb-row">
+          <div class="pb-col-12 pb-col-md-10">
+            <div style="display: flex">
+              <PbButton
+                v-if="!state.isMobileCell"
+                color="primary"
+                buttonStyle="regular"
+                icon="fas fa-arrow-up fa-rotate-270"
+                @click.native="backFunction"
+              />
+              <h2 style="margin: 0;">
+                {{ title }}
+              </h2>
+            </div>
+            <div
+              v-if="!state.isMobileTablet || state.showSidebar"
+              class="sidebar-content"
+            >
+              <slot name="sidebar" />
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div v-if="!state.showSidebar" class="pb-col-12 pb-col-md-9">
+      <div class="pb-row" style="justify-content: center;">
+        <div
+          class="tool-bar pb-col-11"
+          v-if="!enableToolBar"
+        >
+          <slot name="tool-bar" />
+        </div>
+        <div
+          class="main pb-col-11"
+        >
+          <slot name="main" />
+        </div>
+      </div>
+    </div>
+  </section>
+</template>
+
+<script>
+import PbButton from "@pb/Buttons/Button/Button";
+
+export default {
+  name: "PbLayoutGrid",
+  components: {
+    PbButton,
+  },
+
+  props: {
+    title: { type: [String, Number], default: "" },
+    enableToolBar: { type: Boolean, default: false },
+    backFunction: { type: Function, default: () => () => {} },
+  },
+
+  data() {
+    return {
+      state: {
+        isMobileTablet: window.innerWidth <= 768,
+        isMobileCell: window.innerWidth <= 375,
+        showSidebar: false,
+      },
+    };
+  },
+
+  mounted() {
+    window.addEventListener("resize", this.getWidthSizeTablet);
+    window.addEventListener("resize", this.getWidthSizeCell);
+  },
+
+  beforeDestroy() {
+    window.removeEventListener("resize", this.getWidthSizeTablet);
+    window.removeEventListener("resize", this.getWidthSizeCell);
+  },
+
+  methods: {
+    getWidthSizeTablet(e) {
+      const { target } = e;
+
+      this.state.isMobileTablet = target.innerWidth <= 768;
+    },
+
+    getWidthSizeCell(e) {
+      const { target } = e;
+
+      this.state.isMobileCell = target.innerWidth <= 375;
+    },
+
+    toggleSidebar() {
+      this.state.showSidebar = !this.state.showSidebar;
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.layout-grid-container {
+  .sidebar {
+    border-right: solid #eeeeee 1px;
+    margin: 28px 0px 40px;
+
+    .sidebar-content {
+      margin-top: 40px;
+      min-height: 560px;
+    }
+  }
+
+  .tool-bar {
+    height: 40px;
+    margin-top: 28px;
+  }
+
+  .main {
+    min-height: 512px; 
```

Comment:
Shouldn't this slot be able to expand to the full height of the layout?

Reply:
sim, ele vai fazer isso mas ele vai ter uma altura min de 512px